packages/cardano-services/README.md

```diff
@@ -1,7 +1,92 @@
 # Cardano JS SDK | Cardano GraphQL Services
+Libraries and program entrypoints for services that facilitate remote data and submit access using 
+[Provider] interfaces over HTTP, with _optional_ queue-based transaction submission; The 
+[TxSubmitHttpService] can be configured to submit directly via [Ogmios], or via a RabbitMQ broker,
+with one or more workers handling submission and response job creation.
+
+## Features
+- [CLI] or run scripts for the [HTTP server](./src/run.ts) and [worker](./src/startWorker.ts) with 
+configuration via environment variables and _optional_ loading of secrets from disk.
+- Service port discovery via DNS resolution, or static configuration.
+- Fault-tolerant transaction submission via persistent queue, or direct submission.
+- _Optional_ [Prometheus] metrics available at `/metrics`
+
+## Services
+### HTTP Server
+The HTTP server can be started with one or more provider modules by name, segmented by URL path. 
+Run the [CLI] with `start-server --help` to see the full list of options, or inspect the 
+environment variables within [./src/run.ts](./src/run.ts). 
```

Comment:
That file doesn't have any comments about what does each environment variable actually mean. I think it would be very useful to have them listed here with descriptions and default values.

Reply:
Rendering in MD would be nice, but for now, I can more easily add some links to our hosted docs, which are automatically generated and deployed: https://input-output-hk.github.io/cardano-js-sdk/interfaces/_cardano_sdk_cardano_services.HttpServerOptions.html.

Reply:
I don't think autogenerated docs can get to the level of usability as hand-crafted ones. Currently they still don't include any description about what each option mean, how it's translated to environment variables and what would be a sensible way to set them.

Reply:
This will be covered by the task mentioned here: https://github.com/input-output-hk/cardano-js-sdk/pull/348#discussion_r936281539